### DISCUSSION TITLE
List update stuck, exclamation mark with "updated two weeks ago"???

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#3005] [General]
POL-0 list, Windows, Chrome, local uBO build. uBO was not enabled for a long time. I turned in on yesterday. Noticed it today. Purge / Shift purge did not help. What fixed it was to turn off the list, apply and turn it back on. There is `! Last modified: 19 November 2023` in list header, but it's also true for POL-2 and no issue with this one. I don't know how to reproduce - turn uBO off and wait a few weeks?

**Comment by garry-ut99** [comment]
Me too have experienced problems recently with a filter list updating : https://github.com/uBlockOrigin/uAssets/issues/12214#issuecomment-1828347684, 
I'm not sure whether it's an expected/normal behavior of uBO : 
- to need to force a manual update 2 times in a row, if the previous last update was a differential update, 
- also after the filter list updated finally, list header was still outdated

(Also I remember gwarser's old issue https://github.com/uBlockOrigin/uBlock-issues/issues/1165.)

**Comment by gorhill** [comment]
Could be related to the code which will discard an update if it's seen as older than the current cached list. All this code has added complexity and it's difficult to understand if this could be the issue. Maybe that list was last updated in a dev version of uBO for which the `resourceTime` value was not fully settled?

**Comment by krystian3w** [comment]
Same with Firefox 115 ESR (not reproduced yet in 116-122) so not Blink specific.

Workarounds can be:
- backup settings and test:
   - first import from backup, 
   - later reset and import,
   - at end add uBo link store to bookmarks, remove uBo, safe close browser (later reopen), install uBo, and now import

Disable POL-0 may not worked at my side.
